brainscore_language/plugins/pereira2018/data_packaging.py

```diff
@@ -0,0 +1,171 @@
+import logging
+import numpy as np
+import os
+import re
+import scipy.io
+import scipy.stats
+import sys
+from pathlib import Path
+from tqdm import tqdm
+
+from brainio.assemblies import NeuroidAssembly, walk_coords, merge_data_arrays
+from brainscore_language.utils.s3 import upload_data_assembly
+
+_logger = logging.getLogger(__name__)
+
+"""
+The code in this file was run only once to initially upload the data, and is kept here for reference.
+"""
+
+
+def upload_pereira2018(atlas):
+    assembly = load_pereira2018(atlas=atlas)
+    upload_data_assembly(assembly,
+                         assembly_identifier=f"Pereira2018.{atlas}")
```

Comment:
ok, this kind of gets at my earlier comment regarding how we structure assemblies and different networks. this works, but there are caveats. also, can we change "atlas" to "network"? going forward we are not going to use "atlas" but maybe for consistency with the 2021 paper, we can keep atlas here?

Reply:
up to you, don't feel strongly either way

Reply:
#43